demo/admin/src/products/ManufacturerForm.tsx

```diff
@@ -227,54 +232,81 @@ export function ManufacturerForm({ id }: FormProps): React.ReactElement {
                                 name="address.country"
                                 label={<FormattedMessage id="manufacturer.address.country" defaultMessage="Address Country" />}
                             />
-                            <FieldSet
-                                collapsible={true}
-                                initiallyExpanded={false}
-                                title={<FormattedMessage id="manufacturer.address.alternativeAddress" defaultMessage="Alt-Address" />}
-                                supportText={
-                                    <FormattedMessage
-                                        id="manufacturer.address.alternativeAddress.supportText"
-                                        defaultMessage="An alt-address to be used"
+                            <Divider sx={{ marginBottom: 5 }} />
+                            <Field
+                                fullWidth
+                                name="useAlternativeAddress"
+                                type="checkbox"
+                                label={<FormattedMessage id="manufacturer.address.useAlternativeAddress" defaultMessage="Use alternative address" />}
+                            >
+                                {(props) => (
+                                    <FormControlLabel
+                                        label={
+                                            values?.useAlternativeAddress ? (
```

Comment:
Please don't introduce this "no"/"yes" based on the switch value in this PR. This is a new pattern (which I don't think we should use) that is not related to the conditional fields.

(And the interesting thing is that you implemented two different ways to access the useAlternativeAddress value)

Reply:
> Please don't introduce this "no"/"yes" based on the switch value in this PR. This is a new pattern (which I don't think we should use) that is not related to the conditional fields.

I didn't introduce this, I found it in the comet docs: https://storybook.comet-dxp.com/?path=/story/stories-form-finalform-fields--finalformswitch

> And the interesting thing is that you implemented two different ways to access the useAlternativeAddress value

If I had an idea how to combine the `<Field name="useAlternativeAddress" subscription={{` stuff with the `FormControlLabel`-label I would use the Field-subscription and remove the subscription on form-level.

Reply:
> I found it in the comet docs:

that looks like a dev story which is not part of the docs (yes, a confusing mix in our storybook)

Do you have a screendesign that has this label?

> If I had an idea how to combine

I'm not sure what your problem is, but can't you use `props.input.value`?

Reply:
> Do you have a screendesign that has this label?

<img width="1148" alt="image" src="https://github.com/vivid-planet/comet/assets/1324250/00e25de9-8e82-45e7-acc0-0fdec3f531ba">

> I'm not sure what your problem is, but can't you use props.input.value?

I'll try this.

Reply:
what about the '"no"/"yes" based on the switch value'? Should we keep it this way or do you have another idea how to achieve the same?